src/ServiceManagement/ManagedCache/Commands.ManagedCache/PSCacheClient.cs

```diff
@@ -284,6 +284,14 @@ public CloudServiceResource GetNamedCache(string cacheServiceName, string namedC
             return cacheResource;
```

Comment:
Can you please add a message about deprecation into the Changelog.MD in your cmdlets directory?